### PR TITLE
Add blogpost on psf grant

### PR DESCRIPTION
**Pull request by juanis2112**

### CODE REVIEW

File: posts/2021/04/a-step-towards-educating-with-Spyder.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+out of Spyder by applying it in their day-to-day jobs.
+ 
+The first two workshops will cover aspects such as data exploration and 
+visualization with Spyder’s variable explorer and plots panes, getting 
+documentation through Spyder’s help pane, writing good quality and efficient 
+code using Spyder’s code analysis and profiler, etc.
+
+Our last workshop will demonstrate how to create a plugin for Spyder, which, 
+thanks to our new API in [Spyder 5](https://github.com/spyder-ide/spyder/releases/tag/v5.0.0), 
+released in the last few days, will allow users to easily customize and
```
Comment (melissawm) [inline posts/2021/04/a-step-towards-educating-with-Spyder.md:58]: I'd mention either the date of the release, or "released in April 2021".

**Review by melissawm — COMMENTED** [review]
Very nice, thanks @juanis2112 ! It would be nice to add tags if you can - let me know if you want help with that. On my end, this is good to go. As previously discussed with @trallard we want this to be released as soon as possible to time it closer to the recent release of Spyder 5, so no need to wait for the regular blog calendar.

File: posts/2021/04/a-step-towards-educating-with-Spyder.md (hunk 1 above)
Comment (juanis2112) [reply posts/2021/04/a-step-towards-educating-with-Spyder.md]: Agreed. I added some tags and fixed this part. Thanks!

File: posts/2021/04/a-step-towards-educating-with-Spyder.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+For the past months, we have been wondering with the team whether Spyder 
+could also serve as a teaching-learning platform, especially in this era 
+where remote instruction has become necessary. We submitted a proposal to the 
+Essential Open Source Software for Science (EOSS) program of the Chan 
+Zuckerberg Initiative, during its third cycle, with the idea of providing a 
+simple way inside Spyder to create and share interactive tutorials on topics 
+relevant to scientific research. Unfortunately, we didn’t get this funding, 
+but we didn’t let this great idea die.
+
+<!-- TEASER_END -->
```
Comment (rgommers) [inline posts/2021/04/a-step-towards-educating-with-Spyder.md:27]: I suggest moving this to after the next paragraph, to get the PSF grant mention into the teaser.

Comment (juanis2112) [reply posts/2021/04/a-step-towards-educating-with-Spyder.md]: Sounds good @rgommers. Thanks :)

**Review by rgommers — APPROVED** [review]
LGTM, thanks Juanita.

**Comment by rgommers** [comment]
Since @melissawm was already happy, I'll go ahead and merge this.

**Comment by melissawm** [comment]
Thanks, @rgommers ! And thanks @juanis2112 !

**Comment by ccordoba12** [comment]
Yep, thanks @juanis2112, it's a great blogpost!

**Pull request merged**